www/book.js

```diff
@@ -85,7 +85,34 @@ function setup_close() {
 
 window.addEventListener("DOMContentLoaded", setup_close);
 
+function gen_uuid() {
```

Comment:
If you C&P this from elsewhere and you still can find where, good to put a URL in a comment.

Reply:
Hehe… that was generated by GPT-4. I read through it to understand what it does though, of course.

Reply:
Got it.